packages/mask/src/plugins/MaskBox/hooks/useContext.ts

```diff
@@ -135,17 +137,18 @@ function useContext(initialState?: { boxId: string }) {
 
     const boxState = useMemo(() => {
         if (errorMaskBoxInfo || errorMaskBoxStatus || errorBoxInfo) return BoxState.ERROR
-        if (loadingMaskBoxInfo || loadingMaskBoxStatus || loadingBoxInfo) return BoxState.UNKNOWN
+        if (loadingMaskBoxInfo || loadingMaskBoxStatus || loadingBoxInfo) {
+            if (!maskBoxInfo && !boxInfo) return BoxState.UNKNOWN
```

Comment:
When we already know something,  don't lie.

When call retry callback, UI won't flicker.